diff -r 1f62807fdbf4 COPYING
--- a/COPYING	Thu Mar 10 16:54:26 2011 -0500
+++ b/COPYING	Mon Jun 18 14:08:20 2012 +0200
@@ -1,11 +1,15 @@
+TEST 
+
 All source files are Copyright (c) 2008 The Board of Trustees of The
 Leland Stanford Junior University
 
+PATCH TEST
+
 We are making the OpenFlow specification and associated documentation
 (Software) available for public use and benefit with the expectation
 that others will use, modify and enhance the Software and contribute
 those enhancements back to the community. However, since we would like
 to make the Software available for broadest use, with as few
 restrictions as possible permission is hereby granted, free of charge,
 to any person obtaining a copy of this Software to deal in the Software
 under the copyrights without restriction, including without limitation
diff -r 1f62807fdbf4 controller/controller.c
--- a/controller/controller.c	Thu Mar 10 16:54:26 2011 -0500
+++ b/controller/controller.c	Mon Jun 18 14:08:20 2012 +0200
@@ -18,22 +18,23 @@
  * included in all copies or substantial portions of the Software.
  *
  * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND,
  * EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF
  * MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND
  * NONINFRINGEMENT.  IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS
  * BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN
  * ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN
- * CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
- * SOFTWARE.
- *
- * The name and trademarks of copyright holder(s) may NOT be used in
+  Removed 
  * advertising or publicity pertaining to the Software or any
  * derivatives without specific, written prior permission.
+ 
+ **** TEST PATCH
+  Just to see if the patch works
+  
  */
 
 // MAH: start
 // if defined, controller will manage MPLS switches
 #define MPLS_CONTROLLER
 // MAH: end
 
 #include <config.h>
